CMakeLists.txt

```diff
@@ -71,7 +71,12 @@ if(WIN32)
   endif()
 endif()
 
-include_directories("lib" "Descent3" ${PLATFORM_INCLUDES})
+include_directories(
+	${PROJECT_SOURCE_DIR}
```

Comment:
add `ddebug` to the included directories to avoid changing every header (same idea as https://github.com/DescentDevelopers/Descent3/pull/91#discussion_r1573806961)